src/apps/lwaftr/dump.lua

```diff
@@ -36,6 +36,7 @@ function dump_configuration(lwstate)
    local etharr = set('aftr_mac_b4_side',  'aftr_mac_inet_side', 'b4_mac',  'inet_mac')
    local ipv4arr = set('aftr_ipv4_ip')
    local ipv6arr = set('aftr_ipv6_ip')
+   -- TODO: Skip compiled filters
```

Comment:
This `TODO` does not seem to be related to the rest of the patch (or maybe I am missing something here?)